src/jit/hwintrinsiccodegenxarch.cpp

```diff
@@ -593,17 +598,6 @@ void CodeGen::genSSEIntrinsic(GenTreeHWIntrinsic* node)
 
     switch (intrinsicID)
     {
-        case NI_SSE_ConvertScalarToVector128Single:
```

Comment:
Just to confirm. This, and the below, are now considered table driven, correct?

Reply:
Yes.